### PR TITLE
Updating TR translations

### PR DESCRIPTION
**Pull request by kutlay1653**

### CODE REVIEW

**Comment by fire855** [comment]
Can you please resolve the conflicts? Then I can also merge this :)

**Comment by kutlay1653** [comment]
can you check it again? Thanks @fire855

File: packages/apps/Settings/res/values-tr/mad_strings.xml [hunk 1]
```diff
@@ -317,6 +324,30 @@
     <string name="menu_hide_substratum_icons">Hide icon overlays</string>
 
     <!-- Force authorize substratum packages -->
+    <string name="force_authorize_substratum_packages_title">Tüm uygulamaları temalarken doğrulamayı zorla</string>
```
Comment (fire855) [inline packages/apps/Settings/res/values-tr/mad_strings.xml]: Here you start duplicating lines.
"force_authorize_substratum_packages_title" is already in the file.
Can you please remove the lines with english language?

**Comment by fire855** [comment]
There are still duplicates.
You are starting here with adding duplicates: https://github.com/MediatekAndroidDevelopers/mados_custom_strings/pull/166/files#diff-f4e71da65a4d4a86ac934b4b0def8c8dR328

**Comment by fire855** [comment]
@kutlay1653 
Can you fix the duplicates problem? Then I can merge this PR.

**Pull request merged**